json_schemas/framework-agreement-details.json

```diff
@@ -15,6 +15,57 @@
       "minLength": 1,
       "type": "string"
     },
+    "frameworkRefDate": {
+      "minLength": 1,
+      "type": "string",
+      "patterm": "^[0-9-]+$"
+    },
+    "frameworkStartDate": {
+      "minLength": 1,
+      "type": "string",
+      "patterm": "^[0-9/]+$"
+    },
+    "frameworkEndDate": {
+      "minLength": 1,
+      "type": "string",
+      "patterm": "^[0-9/]+$"
```

Comment:
Also, maybe this should be using the built-in `date-time` format (https://spacetelescope.github.io/understanding-json-schema/reference/string.html)